crypto/encode_decode/decoder_meth.c

```diff
@@ -24,6 +24,16 @@
  */
 #define NAME_SEPARATOR ':'
 
+static void ossl_decoder_free(void *data)
+{
+    OSSL_DECODER_free((OSSL_DECODER *)data);
```

Comment:
I notice elsewhere where this is done there is no need to do the cast.

Reply:
I think C++ will whine about the implicit cast.

Reply:
I don't have strong opinions. I think it is just a habit of mine probably because of c++ development. I'll do it without casting.